public/javascripts/SVValidate/src/modal/ModalMissionComplete.js

```diff
@@ -11,7 +11,7 @@ function ModalMissionComplete (uiModalMissionComplete, user, confirmationCode) {
             // Load the audit page since they've done 2 missions.
             window.location.replace('/audit');
         } else {
-            self.hide();
+	    self.hide();
```

Comment:
I think you accidentally removed some space characters and replaced them with tab characters. Could you revert this change? Could you also go over all the new code you added and make sure you are using spaces instead of tabs?